openmc/data/njoy.py

```diff
@@ -323,19 +355,21 @@ def make_ace(filename, temperatures=None, ace='ace', xsdir='xsdir', pendf=None,
             commands += _TEMPLATE_ACER.format(**locals())
 
             # Indicate tapes to save for each ACER run
-            tapeout[nace] = fname.format(ace, temperature)
-            tapeout[ndir] = fname.format(xsdir, temperature)
+            tapeout[nace] = fname.format("ace", temperature)
+            tapeout[ndir] = fname.format("xsdir", temperature)
     commands += 'stop\n'
     run(commands, tapein, tapeout, **kwargs)
 
     if acer:
+        ace = (output_dir / "ace") if acer is True else Path(acer)
```

Comment:
The `open` function only works on path-like objects starting in Python 3.6, so you need an explicit `str(ace)` for the call for it to work on Python 3.5 and before.

Reply:
Just caught this error as well. I was so sure to make sure the `os` calls didn't take a `Path` argument. Will have a fix pushed shortly